javascript/jit_plotter_labels2colors.js

```diff
@@ -0,0 +1,72 @@
+autowatch = 1;
```

Comment:
I would rename this file `fluid.jit.plotter.labels2colors.js` to keep it namespaced and consistent.